script/import-all-data

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# abort on any errors
+set -e
+
+# check that we are in the expected directory
+cd `dirname $0`/..
+
+COMMANDS=$(find hub/management/commands/ -name 'import_*' -printf '%f\n' | sed 's#[.]py##')
+
+for COMMAND in $COMMANDS
+do
+    ./manage.py "$COMMAND"
```

Comment:
I would suggest that this needs to do something to guarantee that `import_areas` is run first as the other commands will not work unless that has run. I suspect explicitly running that first and then do the find thing but with an if so skip if `COMMAND` is `import_areas` is the best way.